src/main/java/com/kuit/agarang/domain/playlist/service/PlaylistService.java

```diff
@@ -0,0 +1,65 @@
+package com.kuit.agarang.domain.playlist.service;
+
+import com.kuit.agarang.domain.memory.model.dto.MemoryDTO;
+import com.kuit.agarang.domain.memory.model.entity.Memory;
+import com.kuit.agarang.domain.memory.repository.MemoryRepository;
+import com.kuit.agarang.domain.playlist.model.dto.MusicUrlsResponse;
+import com.kuit.agarang.domain.playlist.model.dto.PlaylistDto;
+import com.kuit.agarang.domain.playlist.model.dto.PlaylistResponse;
+import com.kuit.agarang.domain.playlist.model.dto.PlaylistsResponse;
+import com.kuit.agarang.domain.playlist.model.entity.MemoryPlaylist;
+import com.kuit.agarang.domain.playlist.model.entity.Playlist;
+import com.kuit.agarang.domain.playlist.repository.MemoryPlaylistRepository;
+import com.kuit.agarang.domain.playlist.repository.PlaylistRepository;
+import lombok.RequiredArgsConstructor;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Service
+@RequiredArgsConstructor
+public class PlaylistService {
+
+    private final PlaylistRepository playlistRepository;
+    private final MemoryPlaylistRepository memoryPlaylistRepository;
+    private final MemoryRepository memoryRepository;
+
+    public PlaylistsResponse getAllPlaylists() {
+        List<Playlist> playlists = playlistRepository.findAll();
+        List<PlaylistDto> playlistDtos = playlists.stream()
+                .map(playlist -> new PlaylistDto(playlist.getId(), playlist.getName(), playlist.getImageUrl()))
+                .collect(Collectors.toList());
+
+        return new PlaylistsResponse(playlistDtos);
+    }
```

Comment:
```suggestion
public List<PlaylistDto> getAllPlaylists() {
    List<Playlist> playlists = playlistRepository.findAll();
    return playlists.stream().map(PlaylistDto::from).toList();
}
```

PlaylistDto 클래스에서 static 메서드로 form 을 생성하고, 반환 값을 List 로 둔다면 이런 코드도 가능합니다.

PlaylistDto.java

```java
public class PlaylistDto {
    private Long id;
    private String name;
    private String imageUrl;

    @Builder
    public PlaylistDto(Long id, String name, String imageUrl) {
        this.id = id;
        this.name = name;
        this.imageUrl = imageUrl;
    }

    public static PlaylistDto from(Playlist playlist) {
        return PlaylistDto.builder()
          .id(playlist.getId())
          .name(playlist.getName())
          .imageUrl(playlist.getImageUrl())
          .build();
    }
}
```